commit first_commit
Author: Dmitriy Karachev <karachevda@yandex-team.ru>
Date:   Sat Dec 22 13:15:43 2018 +0300

    feat tariffs: remove handler from py (#12260)

    Tests: Проверял в unstable.
    Relates: TAXIBACKEND-24790

diff --git a/taxi-utils/utils/protocol_1x/__init__.py b/taxi-utils/utils/protocol_1x/__init__.py
index f411de1..b4b875b 100644
--- a/taxi-utils/utils/protocol_1x/__init__.py
+++ b/taxi-utils/utils/protocol_1x/__init__.py
@@ -148,6 +148,7 @@
+from . import promocode_series
 from . import promocode_usages2
